sites/fast-website/src/public/index.html

```diff
@@ -6,127 +6,114 @@
     <title><%= htmlWebpackPlugin.options.title %></title>
   </head>
   <body>
-    <fast-design-system-provider id="body-provider" class="body-provider"
+    <fast-design-system-provider
+      class="body-provider"
       use-defaults
-      background-color="#1F1F1F"
-      accent-base-color="#F33378"
+      background-color="#181818"
+      accent-base-color="#DA1A5F"
+      type-ramp-base-font-size="16px"
+      type-ramp-base-line-height="1.5"
     >
+      <background-design></background-design>
       <div class="wrapper">
-        <!-- header:start -->
         <header class="header">
           <site-navigation>
             <fast-anchor class="logo" slot="start" appearance="lightweight" href="#">
               <%= require("svg/icon-brand.svg") %>
             </fast-anchor>
             <div class="anchor-wrapper">
               <ul>
-                <li><fast-anchor appearance="lightweight" href="#">Components</fast-anchor></li>
+                <li><fast-anchor appearance="lightweight" href="https://explore.fast.design/">Components</fast-anchor></li>
```

Comment:
This links us to the react version of our components, is that what we want for now?

Reply:
I haven't been given a sitemap so I'm basing these links off of what's on the current live site.

Reply:
The explorer will be updated at the same time this site goes live, so this is fine.